### ISSUE TITLE
NullObserver taking up large amount of time

### ISSUE THREAD

**Issue by jarbus** [twillkens/CoEvo#4]
![image](https://github.com/twillkens/CoEvo/assets/42819002/59fca35c-a451-4e9e-a40b-2d0690d275d8)

get_individual_outcomes and get_observations are taking up a large amount of time when using large population sizes. Wondering if there is a way to avoid the NullObserv{er,ation} and just not run any observation code entirely.

**Comment by twillkens** [comment]
Yep, that can be done. Will be working with observers today when implementing MODES for function graphs and can clean that up.
